gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/GradleUtil.java

```diff
@@ -92,6 +94,18 @@ public static JavaProject convertGradleProject(Project gradleProject) {
         .build();
   }
 
+  private static File extractLocalRepositoryDir(List<String> compileClassPathElements) {
+    String anyLocalArtifact = compileClassPathElements.stream()
+        .filter(c -> c.contains(".gradle"))
+        .findFirst().orElse(null);
+    if (anyLocalArtifact != null) {
+      String[] pathParts = anyLocalArtifact.split(File.separator);
+      String[] localRepoDirPathParts = Arrays.copyOf(pathParts, pathParts.length - 5);
+      return new File(String.join(File.separator, localRepoDirPathParts));
+    }
+    return null;
```

Comment:
Have you tested this in a multi-module project?
What's that variable exactly used for?

Reply:
This was added to be used by JKubeArtifactResolverService which was removed in #1127 

This is should not be required now. Let me remove it.

Reply:
So is that `localRepositoryBaseDirectory` field used anywhere?

Reply:
I'm not able to see any usages of this field now. Earlier it was used in JKubeArtifactResolverService:

https://github.com/eclipse/jkube/blob/3455471f7492508ce6ec220642b31d8339e3b8fd/jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/JKubeArtifactResolverService.java#L38

But it's no longer in the project

Reply:
Maybe we can remove it altogether

Reply:
Shall I create an issue about it?

Reply:
I think you can just remove it in scope of this PR (if it was used by the artifact resolver, it should have been removed then #1127).
Just create a separate commit for the removal of the unused field (in case we need to revert)